data.js

```diff
@@ -776,21 +793,78 @@ const municipiosObj = {
   ],
   "Zacapa": [
     "Zacapa",
-    "Cabañas",
     "Estanzuela",
-    "Gualán",
-    "Huité",
-    "La Unión",
     "Río Hondo",
-    "San Diego",
-    "San Jorge",
+    "Gualán",
     "Teculután",
     "Usumatlán",
+    "Cabañas",
+    "San Diego",
+    "La Unión",
+    "Huité",
+    "San Jorge",
   ],
 }
+/** # Población por departamento
+ * Devuelve como resultado un entero con la catidad total de la población por departamento, sin diacríticos.
+ */
+const poblacionDepto = {
+  "altaverapaz": "1_215_038",
+  "bajaverapaz": "299_476",
+  "chimaltenango": "615_776",
+  "chiquimula": "415_063",
+  "elprogreso": "176_632",
+  "escuintla": "733_181",
+  "guatemala": "3_015_081",
+  "huehuetenango": "1_170_669",
+  "izabal": "408_688",
+  "jalapa": "342_923",
+  "jutiapa": "488_395",
+  "peten": "545_600",
+  "quetzaltenango": "799_101",
+  "quiche": "949_261",
+  "retalhuleu": "326_828",
+  "sacatepequez": "330_469",
+  "sanmarcos": "1_032_277",
+  "santarosa": "396_607",
+  "solola": "421_583",
+  "suchitepequez": "554_695",
+  "totonicapan": "418_569",
+  "zacapa": "245_374",
+}
+
+/** # Población por departamento
+ *  Devuelve como resultado un entero con la catidad total de la población por departamento
+ */
+const poblacionDeptoObj = {
+  "Alta Verapaz": "1_215_038",
+  "Baja Verapaz": "299_476",
+  "Chimaltenango": "615_776",
+  "Chiquimula": "415_063",
+  "El Progreso": "176_632",
+  "Escuintla": "733_181",
+  "Guatemala": "3_015_081",
+  "Huehuetenango": "1_170_669",
+  "Izabal": "408_688",
+  "Jalapa": "342_923",
+  "Jutiapa": "488_395",
+  "Petén": "545_600",
+  "Quetzaltenango": "799_101",
+  "Quiché": "949_261",
+  "Retalhuleu": "326_828",
+  "Sacatepéquez": "330_469",
+  "San Marcos": "1_032_277",
+  "Santa Rosa": "396_607",
+  "Sololá": "421_583",
+  "Suchitepéquez": "554_695",
+  "Totonicapán": "418_569",
+  "Zacapa": "245_374",
+}
```

Comment:
The numbers of keys still are "Strings." They should be "Number."

Reply:
Solve